### PR TITLE
file-converter: Add version 1.2.3

### PR DESCRIPTION
**Pull request by integzz**

### CODE REVIEW

**Comment by integzz** [comment]
Oh, I forgot to write some descriptions. So, I add it below:


File Converter is a very simple tool that allows you to convert and compress one or several file(s) using the context menu in windows explorer.


Its official website is [File Converter](https://file-converter.org/)

**Comment by integzz** [comment]
Is there any bug that I need to fix?

File: bucket/file-converter.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "url": "https://github.com/Tichau/FileConverter/releases/download/v1.2.3/FileConverter-1.2.3-x64-setup.msi",
+    "hash": "2410944075ed4f0bfd4740e9ee2fd17eae4922980937d738c42bed07ac130c32",
+    "extract_dir": "File Converter",
+    "shortcuts": [["FileConverter.exe", "File Converter"]],
+    "checkver": {
+        "url": "https://github.com/Tichau/FileConverter/releases/download/",
+        "re": "FileConverter-([\\d.]+)-x64-setup.msi"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Tichau/FileConverter/releases/download/v$version/FileConverter-$version-x64-setup.msi"
```
Comment (Ash258) [inline bucket/file-converter.json:15]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/Tichau/FileConverter/releases/download/v$version/FileConverter-$version-x64-setup.msi"
            },
            "32bit": {
                "url": "https://github.com/Tichau/FileConverter/releases/download/v$version/FileConverter-$version-x86-setup.msi"
            }
        }
```

File: bucket/file-converter.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "description": "Convert & compress everything in 2 clicks!",
+    "license": "GPL-3.0",
+    "version": "7.2.5.32097",
+    "url": "https://github.com/Tichau/FileConverter/releases/download/v1.2.3/FileConverter-1.2.3-x64-setup.msi",
+    "hash": "2410944075ed4f0bfd4740e9ee2fd17eae4922980937d738c42bed07ac130c32",
+    "extract_dir": "File Converter",
+    "shortcuts": [["FileConverter.exe", "File Converter"]],
+    "checkver": {
+        "url": "https://github.com/Tichau/FileConverter/releases/download/",
+        "re": "FileConverter-([\\d.]+)-x64-setup.msi"
```
Comment (Ash258) [inline bucket/file-converter.json:12]:
```suggestion
        "github": "https://github.com/Tichau/FileConverter"
```

File: bucket/file-converter.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://file-converter.org/",
+    "description": "Convert & compress everything in 2 clicks!",
+    "license": "GPL-3.0",
+    "version": "7.2.5.32097",
+    "url": "https://github.com/Tichau/FileConverter/releases/download/v1.2.3/FileConverter-1.2.3-x64-setup.msi",
+    "hash": "2410944075ed4f0bfd4740e9ee2fd17eae4922980937d738c42bed07ac130c32",
+    "extract_dir": "File Converter",
+    "shortcuts": [["FileConverter.exe", "File Converter"]],
```
Comment (Ash258) [inline bucket/file-converter.json:9]:
```suggestion
    "shortcuts": [
        [
            "FileConverter.exe",
            "File Converter"
        ]
    ],
```

File: bucket/file-converter.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://file-converter.org/",
+    "description": "Convert & compress everything in 2 clicks!",
+    "license": "GPL-3.0",
+    "version": "7.2.5.32097",
+    "url": "https://github.com/Tichau/FileConverter/releases/download/v1.2.3/FileConverter-1.2.3-x64-setup.msi",
+    "hash": "2410944075ed4f0bfd4740e9ee2fd17eae4922980937d738c42bed07ac130c32",
```
Comment (Ash258) [inline bucket/file-converter.json:7]:
```suggestion
    "version": "1.2.3",
    "description": "Various file converter and compressor",
    "homepage": "https://file-converter.org",
    "license": "GPL-3.0-only",
    "architecture": {
        "64bit": {
            "url": "https://github.com/Tichau/FileConverter/releases/download/v1.2.3/FileConverter-1.2.3-x64-setup.msi",
            "hash": "2410944075ed4f0bfd4740e9ee2fd17eae4922980937d738c42bed07ac130c32"
        },
        "32bit": {
            "url": "https://github.com/Tichau/FileConverter/releases/download/v1.2.3/FileConverter-1.2.3-x86-setup.msi",
            "hash": "00bc76929bc173491f2107b97836c7cb18e0bd52917442cdc743a9c4fd00bf3f"
        }
    },
```

**Comment by integzz** [comment]
I see, thanks for revising. I pay attention to complete context.

**Pull request closed without merging**